### ISSUE TITLE
Premiumizer does not recognize nzb password

### ISSUE THREAD

**Issue by Dominik-1980** [piejanssens/premiumizer#314]
The filename looks like this: filename{{password}}.nzb
Premiumizer makes it look like this: filenamepassword.nzb
and it doesn’t recognize the password in the filename. Download fails.

**Comment by Maik1** [comment]
Yes correct the password is to unzip the download, which Premiumize.me will automatically do.
If I upload a filename{{password}}.nzb to the premiumiz.me webpage downloader, it will download, extract with password and move to cloud. Means the whole support on premiumize.me site is there.
